pyproject.toml

```diff
@@ -250,8 +250,6 @@ module = [
   "brotli.*",
   "fsspec.*",
   "jsbeautifier.*",
-  "mako.*",
-  "minijinja.*",
```

Comment:
Favored ignoring these in-line as more explicit, but on the fence. Happy to revert if others feel differently.